app/services/parse_keywords_service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class ParseKeywordsService
+  def initialize(keywords_file)
+    @keywords_file = keywords_file
+  end
+
+  def call
+    csv_data = CSV.read(@keywords_file.path)
```

Comment:
In the service, we [mostly](https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#services) access variable with accessor method instead of accessing directly
```suggestion
    csv_data = CSV.read(keywords_file.path)
```
  
  For that, we just need to define the `attr_reader` for `keywords_file`
  
  ```
  private

  attr_reader : keywords_file
  ```